pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -337,15 +374,16 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
     defaultValue = "" + WebSocketConfiguration.DEFAULT_WEBSOCKET_REFRESH_DELAY
   )
   private Long configureRefreshDelay(final Long refreshDelay) {
-    if (refreshDelay < DEFAULT_MIN_REFRESH_DELAY || refreshDelay > DEFAULT_MAX_REFRESH_DELAY) {
+    if (refreshDelay < DefaultCommandValues.DEFAULT_MIN_REFRESH_DELAY
+        || refreshDelay > DefaultCommandValues.DEFAULT_MAX_REFRESH_DELAY) {
       throw new ParameterException(
           new CommandLine(this),
           String.format(
-              "refreshDelay must be a positive integer between %s and %s",
-              String.valueOf(DEFAULT_MIN_REFRESH_DELAY),
-              String.valueOf(DEFAULT_MAX_REFRESH_DELAY)));
+              "Refresh delay must be a positive integer between %s and %s",
+              String.valueOf(DefaultCommandValues.DEFAULT_MIN_REFRESH_DELAY),
+              String.valueOf(DefaultCommandValues.DEFAULT_MAX_REFRESH_DELAY)));
```

Comment:
nit: Static import for all the `DefaultCommandValues` references would improve readability.